docs/_includes/ds4/symbols.html

```diff
@@ -70,10 +70,10 @@
     <symbol id="icon-tick-small" viewBox="0 0 24 18">
         <path fill="#0654ba" d="M20.429.001L7.673 12.87 2.481 7.677 0 10.386l7.674 7.45L23.14 2.486 20.43 0z"/>
     </symbol>
-    <symbol id="icon-checkbox-checked" viewBox="0 0 23 24">
+    <symbol id="icon-checkbox-checked" viewBox="0 0 24 24">
```

Comment:
How come we changed the viewbox?

Reply:
I changed it for uniformity (or just OCD) and it didn't seem to make a difference. Let me know if you see any kind of regression because of it, and I can switch it back.